usaspending_api/accounts/tests/integration/test_federal_account_v2.py

```diff
@@ -85,33 +85,64 @@ def fixture_data(db):
     ta3 = baker.make("accounts.TreasuryAppropriationAccount", federal_account=fa3, tas_rendering_label="tas-label-3")
     ta4 = baker.make("accounts.TreasuryAppropriationAccount", federal_account=fa4, tas_rendering_label="tas-label-4")
 
+    dabs100 = baker.make(
+        "submissions.DABSSubmissionWindowSchedule", submission_reveal_date="2017-11-01", submission_fiscal_year=2017
+    )
+    sub101 = baker.make(
+        "submissions.SubmissionAttributes",
+        submission_id="101",
+        reporting_fiscal_year=2017,
+        is_final_balances_for_fy=True,
+        submission_window_id=dabs100.id,
+        reporting_period_start="2017-06-01",
+    )
+    sub102 = baker.make(
+        "submissions.SubmissionAttributes",
+        submission_id="102",
+        reporting_fiscal_year=2017,
+        is_final_balances_for_fy=False,
+        submission_window_id=dabs100.id,
+        reporting_period_start="2017-06-01",
+    )
+    sub103 = baker.make(
+        "submissions.SubmissionAttributes",
+        submission_id="103",
+        reporting_fiscal_year=2017,
+        is_final_balances_for_fy=True,
+        submission_window_id=dabs100.id,
+        reporting_period_start="2017-06-01",
+    )
+
     baker.make(
         "accounts.AppropriationAccountBalances",
         final_of_fy=True,
         treasury_account_identifier=ta0,
         total_budgetary_resources_amount_cpe=1000,
-        submission__reporting_period_start="2017-06-01",
+        # submission__reporting_period_start="2017-06-01",
+        submission=sub101,
     )
+    # Will be filtered out because it's submission's "is_final_balances_for_fy" is False
     baker.make(
         "accounts.AppropriationAccountBalances",
-        final_of_fy=False,  # so filter it out
         treasury_account_identifier=ta0,
         total_budgetary_resources_amount_cpe=100,
         submission__reporting_period_start="2017-03-01",
+        submission=sub102,
     )
     baker.make(
         "accounts.AppropriationAccountBalances",
         final_of_fy=True,
         treasury_account_identifier=ta0,
         total_budgetary_resources_amount_cpe=2000,
-        submission__reporting_period_start="2017-06-01",
+        # submission__reporting_period_start="2017-06-01",
```

Comment:
Did you mean to leave these commented out (two places)? I think they can be removed.

Reply:
I don't even remember commenting these out. I removed the two commented out lines and removed the `submission__reporting_period_start` values from the other AppropriationAccountBalances models since they're not used in the query any more.